client/src/extended_bootstrap_css/bootstrap-wet-fixes-extensions.scss

```diff
@@ -268,25 +268,32 @@ div.label {
   background-color: $primaryColor !important;
   color: $textLightColor !important;
 }
-
-/* to be used with light-background */
-a.btn-ib-light {
-  color: inherit;
+.btn-ib-opened {
+  background-color: $backgroundColor;
+  border: 2px solid $backgroundColor;
+  color: $primaryColor !important;
+  border-radius: $standardBorderRadius;
+  margin: 0px 5px 0px 0px;
 }
+
+/* to be used with dark-background */
 .btn-ib-light {
-  background-color: inherit;
-  border: 2px solid $buttonPrimaryColor;
-  color: inherit;
-  border-radius: $standardBorderRadius; 
+  background-color: $primaryColor;
+  border: 2px solid $backgroundColor;
+  color: $backgroundColor;
+  border-radius: $standardBorderRadius;
+  margin: 0px 5px 0px 0px;
```

Comment:
This margin is specific to the use case in the RPB, right? Please make it some styling local to the RPB use case, the styles here should be for the most generic use case (e.g. same margin defaults as other buttons, which I think is none?) to keep it reusable.